### ISSUE TITLE
improve accessibility around input component

### ISSUE THREAD

**Issue by josephmcg** [Satellite-im/Core-PWA#5102]
- give button in the append/slot area a tabindex of -1
  - reasoning - hitting enter while inside input does the action
- make outline go around the entire input boundary (extend beyond the appends)
- determine whether outline should be applied on mouse navigation/autofocus as well as keyboard navigation

**Comment by molimauro** [comment]
I'd apply the outline only on keyboard navigation, with mouse feels unnatural. What do you think?

**Comment by josephmcg** [comment]
agreed, thanks for your input
